DEVELOPMENT.md

```diff
@@ -93,6 +104,12 @@ If editing existing docs or adding new tutorials, please first read through our
 cleanlab follows the [Black](https://black.readthedocs.io/) code style. This is
 enforced by CI, so please format your code before submitting a pull request.
 
+**Reformat files to Black code style:**
+
+```
+$ black <filename or foldername>
```

Comment:
Similar thing as above, `black cleanlab`. Though maybe this section can be omitted since the pre-commit hook already does this?

Reply:
Omited!